docs/src/tutorials/jump_diffusion.md

```diff
@@ -124,17 +135,10 @@ Using the same `affect!` we build a [`VariableRateJump`](@ref):
 jump = VariableRateJump(rate, affect!)
 ```
 
-To make things interesting, let's copy this jump:
-
-```@example tut3
-jump2 = deepcopy(jump)
-```
-
-so that way we have two independent jump processes. We now couple these jumps
-to the `ODEProblem`:
+We now couple the jump to the `ODEProblem`:
 
 ```@example tut3
-jump_prob = JumpProblem(prob, Direct(), jump, jump2)
```

Comment:
Why drop this? Without it the tutorial only shows examples that use one jump, which may give a misleading impression that one can't have more than one jump?

Reply:
I dropped because I felt the goal of this section (as the title suggests) was to illustrate variable jump in the context of SDEs. Therefore, adding another jump in this section can be confusing to the reader. My first reaction was "Why not just double the rate to achieve the same effect?" It got me thinking that there was some sort of trick that I was not getting.

I think if the intention is to demonstrate the case of multivariate jumps  in SDEs. Why don't we just create another section appropriately titled to illustrate such problems? Then we can just mix the variable and constant jumps from the previous two sections.

I will adjust according to your final say on that.

Reply:
I have pushed a suggestion with this extra section.